server/chat-plugins/usersearch.ts

```diff
@@ -0,0 +1,96 @@
+import {Utils} from '../../lib';
+
+const ONLINE_SYMBOL = ` \u25C9 `;
+const OFFLINE_SYMBOL = ` \u25CC `;
+
+function searchUsernames(target: string, page = false) {
+	const results: {offline: string[], online: string[]} = {
+		offline: [],
+		online: [],
+	};
+	for (const curUser of Users.users.values()) {
+		if (!curUser.id.includes(target) || curUser.id.startsWith('guest')) continue;
+		const escapedName = Utils.escapeHTML(curUser.name);
+		const buttonHTML = `<button class="button" name="send" value="/forcerename ${escapedName}&#10;/j view-usersearch-${target}">Forcerename</button>`;
```

Comment:
Please don't add forcerename buttons to the command, that would be _so_ easy to click on accidentally.

Reply:
That would defeat the point of this

Reply:
Maybe add them to the page, but not the command?

This seems like a good thing to discuss in <<staff>>.

Reply:
They are only on the page?

Reply:
spoke w/ mia, she just misread the code